pipelines/build/openjdk_build_pipeline.groovy

```diff
@@ -144,11 +145,56 @@ def sign(config) {
     }
 }
 
+def listArchives() {
+    return sh(
+            script: """find workspace/target/ | egrep '.tar.gz|.zip'""",
+            returnStdout: true,
+            returnStatus: false
+    )
+            .trim()
+            .split('\n')
+}
+
+def writeMetadata(config, filesCreated) {
+    def buildMetadata = [
+            os     : config.os,
+            arc    : config.arch,
+            variant: config.variant,
+            version: config.javaVersion,
+            tag    : config.parameters.TAG
+    ]
+
+    /*
+    example data:
+    {
+        "os": "linux",
+        "arc": "x64",
+        "variant": "hotspot",
+        "version": "jdk8u",
+        "tag": "jdk8u202-b08",
+        "adopt_build_number": 2,
```

Comment:
Should we make this part of the version number?

Reply:
well, this is a question of how we would bring this into the version that would be compatible with the upstream version number

Reply:
Did you see https://bugs.openjdk.java.net/browse/JDK-8207849 - that might help - although not as of today I guess.